src/widgets/sliderwidget.h

```diff
@@ -99,6 +99,7 @@ class VolumeSlider : public Slider {
  public:
   VolumeSlider(QWidget* parent, uint max = 0);
   QPixmap volumePixmapDraw() const;
+  void VolumeSliderHandle();
```

Comment:
Oh yes, they are accessed only inside public member function of the class.. I can change that